### PR TITLE
Geosafe

### PR DESCRIPTION
**Pull request by lucernae**
Update:
- Add indicator for analysis task process
- Add QML files for InaSAFE Headless
- Hide layer name in qml (because we might have random filename), but add options to override the label name

### CODE REVIEW

**Comment by lucernae** [comment]
@Gustry @ismailsunni you might want to review this, since I made some little changes in geonode/qgis_server

**Comment by ismailsunni** [comment]
Why there is a migration for qgis_server model although you didn't add anything in the model? @lucernae

File: geonode/qgis_server/views.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def download_zip(request, layername):
     return resp
 
 
-def legend(request, layername):
+def legend(request, layername, layertitle=None):
```
Comment (Gustry) [inline geonode/qgis_server/views.py]: Thanks, it's better like this.
How do we use that parameter ? This parameter is not the `urls.py`.

Comment (lucernae) [inline geonode/qgis_server/views.py]: Ah, I forgot to put it in urls.py. :p
So, previously the legend uses the layer filename (which I generated using tmpfile), and it looks ugly, so I hide the label. We also can use layer title on the legend, but I didn't bother searching the view that uses it :p.

**Comment by lucernae** [comment]
I think I'll just go ahed and merge this for now.
I tried putting a layer title in the legend, assuming it can be filled out from this tutorial: http://hub.qgis.org/projects/quantum-gis/wiki/QGIS_Server_Tutorial#GetLegendGraphics.
But apparently it doesn't work.

**Pull request merged**